shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1144,7 +1144,12 @@ public boolean onAccessibilityHoverEvent(MotionEvent event) {
             return false;
         }
 
-        SemanticsNode semanticsNodeUnderCursor = getRootSemanticsNode().hitTest(new float[] {event.getX(), event.getY(), 0, 1});
+        final SemanticsNode rootNode = getRootSemanticsNode();
+        if (rootNode == null) {
```

Comment:
Based on the issue conversation, I can see how this solves the crash. Do you happen to know what the circumstances are that leads to the `rootNode` being null? Is it possible that we have a bug in `FlutterView` or `FlutterActivity` that results in an erroneous state here?

Reply:
It's possible we're dropping state somewhere there but I don't have a reproduction.  I can try to look into that more today

Reply:
@matthew-carroll @dnfield I have uploaded a simple app that reproduces the issue reliably on the bug.

Reply:
Able to reproduce this locally.  What's happening is that FlutterView is resetting the bridge, but the framework isn't sending a full update AFAICT.  Looking at a complete fix for it.  Will close this in the mean time.